odoo_backup_sh/doc/index.rst

```diff
@@ -29,46 +29,45 @@ Configuration
   server_wide_modules = web,odoo_backup_sh
   # (...)
 
-In order for desired databases to appear in a backup dashboard, you have to create settings for it.
-Open the menu ``[[ Backups ]] >> Dashboard``. Click on the ``[Add Database]`` button in the dashboard header.
-Fill required fields:
-
-- Database: select one of your databases
-- Auto Backup Execute Every:  type a value and select a unit of measure.
-  Example: You want the module to make new backup every 2 hours.
-  If you want to create backups manually only, leave this field blank.
-- Next Execution Date: It shows next planned auto backup date. You can correct this.
-  Example: You want the module to make new backup every day at night time.
-- Auto Rotation: If you have set up the auto backup, you can specify how many backups to preserve for certain time
-  frames.
-  Example: The module makes auto backup your database every 2 hours.
-  You want to preserve 2 daily backups and 1 weekly only.
-  Set up Daily and Weekly rotation options as Limited and put the numbers in limit fields. All other options mark as Disabled.
+To appear desired databases in the backup dashboard, you have to set up settings for it.
+
+* Open the menu ``[[ Backups ]] >> Dashboard``.
+* Click on the ``[Add Database]`` button in the dashboard header.
+* Fill required fields:
+
+    * Database: select one of your databases
+
+    * Auto Backup Execute Every:  type a value and select a unit of measure.
+
+*Example: You want the module to make new backup every 2 hours. If you want to create backups manually only, leave this field blank.*
+
+    * Next Execution Date: It shows next planned auto backup date. You can correct this.
+
+*Example: You want the module to make new backup every day at night time.*
+
+    * Auto Rotation: If you have set up the auto backup, you can specify how many backups to preserve for certain time frames.
+
+*Example: The module makes auto backup your database every 2 hours. You want to preserve 2 daily backups and 1 weekly only.Set up Daily and Weekly rotation options as Limited and put the numbers in limit fields.*
+
+* All other options mark as **Disabled**.
 
 After all required fields will be filled, click on the ``[Save]`` button.
 
 All databases with their settings are available in the ``[[ Backups ]] >> Configuration >> Dashboard`` section.
+
 Each database specified in these settings will be presented in the dashboard.
 
 * `Activate Developer Mode <https://odoo-development.readthedocs.io/en/latest/odoo/usage/debug-mode.html>`__
 
 Usage
```

Comment:
Usage section should also contain an insturction how to make manual backup, how to download backup from the cloud